src/GitExtensions.GerritPlugin/GerritUICommands.cs

```diff
@@ -0,0 +1,58 @@
+﻿using GitCommands;
+using GitUI;
+using GitUIPluginInterfaces;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitExtensions.GerritPlugin
+{
+    public sealed class GerritUICommands : IGerritUICommands
+    {
+        private readonly GitUICommands gitUICommands;
+
+        public GerritUICommands(GitUICommands gitUICommands)
+        {
+            this.gitUICommands = gitUICommands;
+        }
+
+        public IGitModule GitModule => gitUICommands.GitModule;
+
+        public ILockableNotifier RepoChangedNotifier { get; }
+
+        public event EventHandler<GitUIPostActionEventArgs> PostCommit;
+        public event EventHandler<GitUIEventArgs> PostRepositoryChanged;
+        public event EventHandler<GitUIPostActionEventArgs> PostSettings;
+        public event EventHandler<GitUIPostActionEventArgs> PostUpdateSubmodules;
+        public event EventHandler<GitUIEventArgs> PostBrowseInitialize;
+        public event EventHandler<GitUIEventArgs> PostRegisterPlugin;
+        public event EventHandler<GitUIEventArgs> PreCommit;
+
+        public void AddCommitTemplate(string key, Func<string> addingText, Image icon) 
+            => gitUICommands.AddCommitTemplate(key, addingText, icon);
+
+        public IGitRemoteCommand CreateRemoteCommand() 
+            => gitUICommands.CreateRemoteCommand();
+
+        public void RemoveCommitTemplate(string key) 
+            => gitUICommands.RemoveCommitTemplate(key);
+
+        public void StartBatchFileProcessDialog(string batchFile) 
+            => gitUICommands.StartBatchFileProcessDialog(batchFile);
+
+        public void StartCommandLineProcessDialog(IWin32Window owner, string command, ArgumentString arguments)
+            => gitUICommands.StartCommandLineProcessDialog(owner, command, arguments);
+
+        public bool StartCommandLineProcessDialog(IWin32Window owner, IGitCommand command) 
+            => StartCommandLineProcessDialog(owner, command);
+
+        public void StartRemotesDialog() 
+            => gitUICommands.RunCommand(new string[] { "ge.exe", "remotes" });
```

Comment:
I'm not sure if this is a valid approach to replace the old behavior

Reply:
but it seems to work

Reply:
I guess it should work without "ge.exe":
```suggestion
            => gitUICommands.RunCommand(new string[] { "remotes" });
```

But do you need this whole file just because I removed this function from the interface `IGitUICommands`?
If yes, I would rather restore the function.
If not, please update the PR title and description.

Reply:
`IGitUICommands.StartRemotesDialog` has been exposed again with gitextensions/gitextensions#8826.